admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/renderer.html

```diff
@@ -38,14 +38,10 @@
 /extensions-->
 
 <script>
-    const VueFormGenerator = window.VueFormGenerator;
-    const VueMultiselect = window.VueMultiselect
-    const VueImagesLoaded = window.VueImagesLoaded
-    const VueTrumbowyg = window.VueTrumbowyg
-    Vue.component('vue-form-generator', VueFormGenerator.component)
-    Vue.component('vue-multiselect', VueMultiselect.default)
+    Vue.component('vue-form-generator', window.VueFormGenerator.component);
+    Vue.component('vue-multiselect', window.VueMultiselect.default);
     Vue.component('datetime', window.VueDatetime.Datetime);
-    Vue.component('Trumbowyg', VueTrumbowyg.default);
-    Vue.directive('images-loaded', VueImagesLoaded)
-    Vue.use(VueFormWizard)
+    Vue.component('Trumbowyg', window.VueTrumbowyg.default);
+    Vue.directive('images-loaded', window.VueImagesLoaded);
+    Vue.use(VueFormWizard);
```

Comment:
why exactly is it `window.XYZ` everywhere except for `VueFormWizard`
I'd prefer NOT to have everything inside the window. That is called "window-object-pollution" and is a bad thing to do

Reply:
Simple omission, fixed.

Reply:
> @reggie7 how exactly do i test the functionality ?

This is a simple cleanup/refactor to allow better readability of code. For checking it's enough to smoke-test a couple of pages to confirm their editing is still possible as before.